ssl-config/src/main/scala/com/avast/sst/ssl/SslContextModule.scala

```diff
@@ -27,6 +30,22 @@ object SslContextModule {
     ).build
   }
 
+  /** Initializes [[javax.net.ssl.SSLContext]] from the provided config if it is enabled.
+    *
+    * Expects a boolean value `enabled` at the root of the provided [[com.typesafe.config.Config]]
+    * which determines whether to initialize the context or not.
+    *
+    * @param withReference Whether we should use reference config of "ssl-config" library as well.
+    */
+  def makeMaybe[F[_]: Sync](config: Config, withReference: Boolean = true): F[Option[SSLContext]] = {
```

Comment:
could we call it `makeIfEnabled`?

Reply:
or at least `makeOption` - we ain't doing haskell here :D

Reply:
The word is not supposed to be the data type, I think "maybe" nicely represents optionality. But `makeIfEnabled` seems more clear so I am for to rename it, thanks.